Tests/Common/TestUtils.cs

```diff
@@ -37,7 +36,7 @@ public static class TestUtils
                 NwkSKey = simulatedDevice.LoRaDevice.NwkSKey,
                 GatewayID = simulatedDevice.LoRaDevice.GatewayID,
                 IsOurDevice = true,
-                ClassType = (simulatedDevice.ClassType is 'C' or 'c') ? LoRaDeviceClassType.C : LoRaDeviceClassType.A,
+                ClassType = simulatedDevice.ClassType == LoRaDeviceClassType.C ? LoRaDeviceClassType.C : LoRaDeviceClassType.A,
```

Comment:
Can't this be just the following?

```suggestion
                ClassType = simulatedDevice.ClassType,
```

Reply:
I think it could, since we don't use class B in any case. I just wanted to do the mechanical change here, though. In any case, I think it's probably worth getting rid of the complexity, so I'll apply the suggestion :)